docs/tsconfig.json

```
@@ -0,0 +1,34 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "~/*": ["./*"]
+    },
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
```

I believe we should strive for 
```suggestion
    "allowJs": false,
```

---

I'm not sure what NextJS would say about it, because this setting was auto-generated here. And we still have some JS files which cannot/shouldn't be converted to TS.

---

I think we still need `allowJs` to be true, some files are not yet moved over to typescript (intentionally). I fully agree that we should aim for that, but we can't do that just yet.